spec/classes/puppet_agent_osfamily_darwin_spec.rb

```diff
@@ -4,58 +4,59 @@
+    Puppet::Parser::Functions.newfunction(:pe_compiling_server_aio_build, type: :rvalue) do |_args|
       master_package_version
     end
   end
 
   package_version = '1.10.100'
 
   facts = {
-    :is_pe                       => true,
-    :osfamily                    => 'Darwin',
-    :operatingsystem             => 'Darwin',
-    :macosx_productversion_major => '10.13',
-    :architecture                => 'x86_64',
-    :servername                  => 'master.example.vm',
-    :clientcert                  => 'foo.example.vm',
-    :env_temp_variable           => '/tmp',
-    :puppet_agent_pid            => 42
+    is_pe: true,
+    osfamily: 'Darwin',
+    operatingsystem: 'Darwin',
+    macosx_productversion_major: '10.13',
+    architecture: 'x86_64',
+    servername: 'master.example.vm',
+    clientcert: 'foo.example.vm',
+    env_temp_variable: '/tmp',
+    puppet_agent_pid: 42
   }
 
   describe 'supported environment' do
-    let(:params) {{ package_version: package_version }}
-    context "when running a supported OSX" do
-      ["osx-10.12-x86_64", "osx-10.13-x86_64", "osx-10.14-x86_64", "osx-10.15-x86_64", "osx-11-x86_64"].each do |tag|
+    let(:params) { { package_version: package_version } }
+
+    context 'when running a supported OSX' do
+      ['osx-10.12-x86_64', 'osx-10.13-x86_64', 'osx-10.14-x86_64', 'osx-10.15-x86_64', 'osx-11-x86_64'].each do |tag|
         context "on #{tag} with no aio_version" do
           let(:osmajor) { tag.split('-')[1] }
 
           let(:facts) do
             facts.merge({
-              :is_pe                       => true,
-              :aio_agent_version           => '1.10.99',
-              :platform_tag                => tag,
-              :macosx_productversion_major => osmajor
-            })
+                          is_pe: true,
+              aio_agent_version: '1.10.99',
+              platform_tag: tag,
+              macosx_productversion_major: osmajor
+                        })
           end
 
-          it { should compile.with_all_deps }
+          it { is_expected.to compile.with_all_deps }
           it { is_expected.to contain_file('/opt/puppetlabs') }
           it { is_expected.to contain_file('/opt/puppetlabs/packages') }
           it { is_expected.to contain_file("/opt/puppetlabs/packages/puppet-agent-#{package_version}-1.osx#{osmajor}.dmg") }
           it { is_expected.to contain_file('/tmp/osx_install.sh') }
           it { is_expected.to contain_exec('osx_install script') }
-          it { is_expected.to contain_class("puppet_agent::osfamily::darwin") }
+          it { is_expected.to contain_class('puppet_agent::osfamily::darwin') }
 
           context 'aio_agent_version is out of date' do
             let(:facts) do
               facts.merge({
-                :aio_agent_version => '0.0.1'
-              })
+                            aio_agent_version: '0.0.1'
+                          })
```

Comment:
And again? If it's a single line it kind of makes sense what/how rubocop wanted to indent but it should be consistent.

Reply:
Probably the `auto_correct` fixes only the first line in the hash